ethereum/migrations/3_add_erc721.js

```diff
@@ -0,0 +1,46 @@
+require("dotenv").config();
```

Comment:
can we rename this migration to 5_...js? or keep it at 3 and rename the others to be later

Reply:
Done